quantum/quantum_keycodes.h

```diff
@@ -104,27 +104,57 @@
 
 // L-ayer M-od: Momentary switch layer with modifiers active - 16 layer max
 #define LM(layer, mod) (QK_LAYER_MOD | (((layer)&0xF) << 5) | ((mod)&0x1F))
+#define IS_LM(kc) ((kc) >= QK_LAYER_MOD && (kc) <= QK_LAYER_MOD_MAX)
```

Comment:
None of these `IS_*` should be introduced as they already exist <https://github.com/qmk/qmk_firmware/blob/master/quantum/keycodes.h#L1276-L1323>

Reply:
Sure, they are a leftover from before these where introduced with the DD keycodes.

Reply:
They are gone now.